packages/web-pkg/src/components/Spaces/QuotaModal.vue

```diff
@@ -73,34 +78,39 @@ export default defineComponent({
       this.selectedOption = option.value
     },
     editQuota() {
-      return this.graphClient.drives
-        .updateDrive(this.space.id, { quota: { total: this.selectedOption } }, {})
-        .then(({ data }) => {
-          this.cancel()
-          if (unref(this.$router.currentRoute).name === 'admin-settings-spaces') {
-            this.$emit('spaceQuotaUpdated', data.quota)
-          }
-          this.UPDATE_SPACE_FIELD({
-            id: this.space.id,
-            field: 'spaceQuota',
-            value: data.quota
-          })
-          this.UPDATE_RESOURCE_FIELD({
-            id: this.space.id,
-            field: 'spaceQuota',
-            value: data.quota
-          })
-          this.showMessage({
-            title: this.$gettext('Space quota was changed successfully')
+      const requests = []
+      this.spaces.forEach((space) => {
+        const request = this.graphClient.drives
+          .updateDrive(space.id, { quota: { total: this.selectedOption } }, {})
+          .then(({ data }) => {
+            this.cancel()
+            if (unref(this.$router.currentRoute).name === 'admin-settings-spaces') {
+              this.$emit('spaceQuotaUpdated', data.quota)
+            }
+            this.UPDATE_SPACE_FIELD({
+              id: space.id,
+              field: 'spaceQuota',
+              value: data.quota
+            })
+            this.UPDATE_RESOURCE_FIELD({
+              id: space.id,
+              field: 'spaceQuota',
+              value: data.quota
+            })
+            this.showMessage({
+              title: this.$gettext('Space quota was changed successfully')
+            })
```

Comment:
When we disable/enable spaces, we only show one message. I think we should keep that consistent, also, displaying 20 messages for 20 updates spaces is not so nice IMO. You can check the `disable.ts` mixin to have a look on how that's done.